pkg/remotewriteutils/util.go

```diff
@@ -25,6 +26,12 @@ func ToRemoteWrite(obj interface{}) (*pmov1alpha1.RemoteWrite, error) {
 		return nil, microerror.Maskf(wrongTypeError, "'%T' is not a 'pmov1alpha1.RemoteWrite'", obj)
 	}
 
+	kind := reflect.TypeOf(remotewrite).Elem().Name()
```

Comment:
I wonder if this is still needed after adding the subresource marker.

Reply:
Good point this might not be needed. But this need to be tested.